### ISSUE TITLE
Not Authorized to Play messages.

### ISSUE THREAD

**Issue by lindner** [resonatecoop/stream2own#31]
### Operating system
MacOS 10.14.3

### Browser name and version
Chrome Stable v77

### Expected behavior

Have balance decrement, not see error popups

### Actual behavior

I somehow have an expired token in my browser session, but I can still listen to music.  I don't see my balance decrementing and I see an toast message `Not authorized to play`.  Reloading by logout/login shows that the tracks are marked as played though and the balance is decremented.

The console log shows the following:

````
index.js:73 POST https://api.resonate.is/v1/users/2367/plays?client_id=ba9f575a-0af8-49f9-84d6-70869ef991dc 401 (Unauthorized)
(anonymous) @ index.js:73
Object.assign.data @ index.js:69
s.<computed> @ index.js:123
(anonymous) @ player.js:47
E @ runtime.js:65
(anonymous) @ runtime.js:303
e.<computed> @ runtime.js:117
n @ player.js:47
(anonymous) @ player.js:47
i @ player.js:47
(anonymous) @ player.js:47
o._emit @ index.js:159
o.emit @ index.js:25
value @ index.js:598
o._emit @ index.js:159
o.emit @ index.js:25
timeupdate @ index.js:53
index.js:147 10:27:17 ✨ store:player User is not authorized to play
index.js:147 10:31:26 ✨ player Next {count: 0, fav: 0, url: "https://api.resonate.is/v1/stream/19084", track: {…}, track_group: Array(1)}
index.js:147 10:31:26 ✨ player Playing
index.js:147 10:31:26 ✨ nanoplayer loaded src
index.js:147 10:31:26 ✨ player Mutating player component
index.js:147 10:31:26 ✨ nanoplayer playing new audio
````

Network tab shows successful request at the start of the track:

`Request URL: https://api.resonate.is/v1/stream/19084?client_id=ba9f575a-0af8-49f9-84d6-70869ef991dc&preview=false`

Followed by a 401 failure

```
Request URL: https://api.resonate.is/v1/users/2367/plays?client_id=ba9f575a-0af8-49f9-84d6-70869ef991dc
Request Method: POST
Status Code: 401 Unauthorized
Remote Address: 46.4.17.130:443
Referrer Policy: no-referrer-when-downgrade
```
*Response Headers*
```
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: https://beta.resonate.is
Cache-Control: no-store, no-cache, must-revalidate, no-store, no-cache
Connection: keep-alive
Content-Type: application/json
Date: Fri, 30 Aug 2019 17:32:12 GMT
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Pragma: no-cache, no-cache
Server: nginx/1.12.2
Transfer-Encoding: chunked
```
*Request Headers*
```
Provisional headers are shown
accept-language: en
content-type: application/json
Referer: https://beta.resonate.is/playlist/latest
Sec-Fetch-Mode: cors
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.19 Safari/537.36
```
*Query String params*
```
client_id: ba9f575a-0af8-49f9-84d6-70869ef991dc
```
*Request Payload*
```
{uid: 2367, tid: 19084, client_id: "ba9f575a-0af8-49f9-84d6-70869ef991dc"}
client_id: "ba9f575a-0af8-49f9-84d6-70869ef991dc"
tid: 19084
uid: 2367
```
*JSON Response*
```
{message: "Unauthorized: Missing access token", status: 401, data: null}
data: null
message: "Unauthorized: Missing access token"
status: 401
```

### Steps to reproduce behavior
Not sure...  let your token expire?

**Comment by auggod** [comment]
Your client id should not be shared publicly, so I assigned a new one to your account. You'll have to login again. 
Token lifetime is quite short. (only 24h) Also, it seems, we're having problems getting a refreshed token in time on the frontend. I'll see what's best to do. Maybe, we should increase token lifetime as it seems these kind of errors are becoming redundant.
You don't need a token to load the audio files (only client id is needed), that's why I had to set a new one.
